extension/Config.jsm

```diff
@@ -0,0 +1,69 @@
+/* to use:
+
+- Recall this file has chrome privileges
+- Cu.import in this file will work for any 'general firefox things' (Services,etc)
+  but NOT for addon-specific libs
+*/
+
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(config|EXPORTED_SYMBOLS)" }]*/
+const EXPORTED_SYMBOLS = ["config"];
+
+// var slug = "shield-example-addon"; // should match chrome.manifest;
+
+const config = {
+  study: {
+    studyName: "share-button-study", // no spaces, for all the reasons
```

Comment:
Overall, you might want to give this file a pass and make sure all the comments are correct and make sense in context.